ginga/rv/plugins/Catalogs.py

```diff
@@ -67,13 +71,15 @@ def __init__(self, fv, fitsimage):
         canvas = self.dc.DrawingCanvas()
         canvas.enable_draw(True)
         canvas.set_drawtype(self.drawtype, color='cyan', linestyle='dash')
-        canvas.set_callback('cursor-down', self.btndown)
-        canvas.set_callback('cursor-up', self.btnup)
         canvas.set_callback('draw-event', self.draw_cb)
+        canvas.add_draw_mode('select', down=self.btndown,
+                             move=self.btndown, up=self.btnup)
```

Comment:
For that matter, it doesn't look like `btndown` is doing anything either, so maybe `down` can also be set to `None` or omitted.